tests/test_all_rules.py

```diff
@@ -455,6 +456,30 @@ def test_required_lookback(self):
             err_msg = f'The following rules should have a longer `from` defined, due to indexes used\n {rules_str}'
             self.fail(err_msg)
 
+    def test_eql_lookback(self):
+        """Ensure EQL rules lookback => max_span, when defined."""
+        unknowns = []
+        invalids = []
+
+        for rule in self.all_rules:
+            if rule.contents.data.type == 'eql' and rule.contents.data.max_span:
+                if rule.contents.data.look_back == 'unknown':
+                    unknowns.append(self.rule_str(rule, trailer=None))
```

Comment:
can we just warn in here instead?

Reply:
we have a lot of rules and bulk changes are not uncommon. Do we really need a full warning wrapper around every instance?

is this
```
tests/test_all_rules.py::TestRuleTiming::test_eql_lookback
  /Users/jibarra/PycharmProjects/detection-rules-fork/tests/test_all_rules.py:466: UserWarning: f036953a-4615-4707-a1ca-dc53bf69dcd5 - Unusual Child Processes of RunDLL32 unable to determine lookback for maxspan check
    warnings.warn(f'{self.rule_str(rule, trailer=None)} unable to determine lookback for maxspan check')

tests/test_all_rules.py::TestRuleTiming::test_eql_lookback
  /Users/jibarra/PycharmProjects/detection-rules-fork/tests/test_all_rules.py:466: UserWarning: 1defdd62-cd8d-426e-a246-81a37751bb2b - Execution of File Written or Modified by PDF Reader unable to determine lookback for maxspan check
    warnings.warn(f'{self.rule_str(rule, trailer=None)} unable to determine lookback for maxspan check')
```

really providing any greater value over this

```
tests/test_all_rules.py::TestRuleTiming::test_eql_lookback
  detection-rules-fork/tests/test_all_rules.py:477: UserWarning: Unable to determine lookbacks for the following rules:
  f036953a-4615-4707-a1ca-dc53bf69dcd5 - Unusual Child Processes of RunDLL32
  1defdd62-cd8d-426e-a246-81a37751bb2b - Execution of File Written or Modified by PDF Reader
    warnings.warn(f'Unable to determine lookbacks for the following rules:\n{warn_str}')
```